lib/cmds/space_cmds/generate_cmds/migration.js

```diff
@@ -0,0 +1,237 @@
+import { getContext } from '../../../context'
+import { assertLoggedIn, assertSpaceIdProvided, assertEnvironmentIdProvided } from '../../../utils/assertions'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { log, success } from '../../../utils/log'
+
+const recast = require('recast')
+const camelCase = require('camelcase')
+const b = recast.types.builders
+const _ = require('lodash')
+const toAst = require('to-ast')
+const prettier = require('prettier')
+const { createClient } = require('contentful-management')
+const fs = require('fs')
+
+export const command = 'migration'
+
+export const desc = 'Generate a migration file for your content model or a specific content type'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful space generate migration')
+    .option('space-id', {
+      describe: 'ID of the space the content model will belong to',
+      type: 'string'
+    })
+    .option('environment-id', {
```

Comment:
you can add `default: 'master'` to the option so when the user does not pass an environment the value will default to `master`